CHANGELOG.md

```diff
@@ -1,3 +1,28 @@
+### citus v12.1.3 (April 18, 2024) ###
+
+* Allows overwriting host name for all inter-node connections by supporting "host"
+  parameter in citus.node_conninfo (#7541)
+
+* Correctly mark some variables as volatile (#7570)
```

Comment:
We use present simple tense to start description entries:

```suggestion
* Fixes some potential bugs by correctly marking some variables as volatile (#7570)
```

Reply:
Added as you suggested